plugin.json

```diff
@@ -21,6 +21,10 @@
       "hook": "action:post.unvote",
       "method": "hooks.actions.postUnvote"
     },
+    {
+      "hook": "action:post.downvote",
```

Comment:
The plugin designed to have only positive actions. It does not try to handle downvotes and subtract operation for the points.

Reply:
It will have a problem if it completely ignores downvoting, if a user does
upvote (+1, +1 pt)
downvote (0, 0 pt)
unvote (0, 0 pt)
it will return to the unvoted state while two people gets point +1.

Reply:
Changed it so there is no negative bonus for downvotes.

Reply:
It was by design (What impactful and positive action you could do to be recognized for). It does not try to handle abusive use from the users. Definitely, do not want to handle all edge cases related to removing the points (Down-level).

Ideally, all point increases should be tracked and if you unvote/vote, you will not get anything and nobody will not lose anything. 

I recommend focussing on all necessary changes for compatibility issues. Behavior changes, feature development are the stories for a different time.